CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.1]
```

Comment:
When a new feature is added, increment the second number. So, this version will be `0.4.0`.

Reply:
Yesterday I created 2 PR, so I thought when one of them will be merged than I'll change the version of another one.